src/app/rundown-view/components/minishelf/mini-shelf.component.html

```diff
@@ -3,17 +3,16 @@
   title="{{ segment.name }}"
   (click)="executeAction()"
 >
-<!--  TODO: add ThumbnailComponent and use it here-->
-    <img
+  <!--  TODO: add ThumbnailComponent and use it here-->
```

Comment:
Minor thing: This comment should be deleted because it is referring to the Tooltip which you now have added as `sofie-mini-shelf-tooltip` (even though it is still missing the tooltip information) :)

Reply:
Done